src/shared/components/mutationTable/column/GenericCategoricalColumnFormatter.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import { Mutation } from 'cbioportal-ts-api-client';
+import _ from 'lodash';
+
+/**
+ * @author Pim van Nierop
+ */
+export default class GenericCategoricalColumnFormatter {
```

Comment:
I'm not sure if this name is apt since it seems to use the namespace terminology

Reply:
Ah you are right. Names update to `NumericNamspaceColumnFormatter` and `CategoricalNamspaceColumnFormatter`.